web/frontend/src/components/keyboard-input.vue

```diff
@@ -74,9 +82,16 @@ const removeKeyboardListeners = () => {
   window.removeEventListener('keyup', onUseKeyboardNav);
 };
 
+const stopBase = () => {
+  const req = new baseApi.StopRequest();
+  req.setName(props.name);
+  window.baseService.stop(req, new grpc.Metadata(), displayError);
```

Comment:
One concern I have here: do we know if this gives an error message if the base has already been stopped? If so, then we could end up giving our peeps error messages when they've done nothing wrong. If this is what's going to end up happenning I'd recommend not passing errors to `displayError`.

Reply:
Based on running this using the fake config, if I toggle the keyboard on, then press the Stop button, and then switch the keyboard toggle to off (which will also kick off the stop command), I see no errors.   However, will this be the case when running on a real robot?...I'm not sure :(.

Reply:
No prob! We can punt on that then.

Reply:
Confirmed, no errors :)